roles/matrix-postgres-backup/defaults/main.yml

```diff
@@ -0,0 +1,140 @@
+matrix_postgres_backaup_enabled: false
+
+matrix_postgres_connection_hostname: "matrix-postgres"
+matrix_postgres_connection_port: 5432
+matrix_postgres_connection_username: "matrix"
+matrix_postgres_connection_password: ""
```

Comment:
These had better be named `matrix_postgres_backup_connection_*` (`matrix_postgres_backup_*` prefix).

Reasons:

1. You're redefining `matrix-postgres` roles variables here. It's confusing
2. Even if you're not really redefining variables from other roles, but rather using them, it's still not ideal. We try to make roles as independent of one another as possible, with `group_vars/matrix_servers` trying to tie their variables together.

That is to say, these variables should have some defaults (or even empty defaults, with `roles/matrix-postgres-backup/tasks/validate_config.yaml` complaining about them being blank). And then `group_vars/matrix_servers` will redefine these variables in term of other variables. Example (excerpt) from `group_vars/matrix_servers`:

```yaml
matrix_postgres_backup_connection_hostname: "{{ matrix_postgres_connection_hostname }}"
# Same for other variables
```

Reply:
Thx. I resued the name by indent as the value should be the same as from the other role. I did not know of the machanism to use `group_vars/matrix_servers` to redefine these variables. I will change this.